include/hatrack/crown.h

```diff
@@ -31,7 +31,7 @@
 #include "mmm.h"
 
 typedef struct {
-    void    *item;
+    alignas(8) void *item;
```

Comment:
This `alignas(8)` does nothing

Reply:
These things did seem to be unaligned in one instance at some point, though the core problem was somewhere else and fixed. I'll clean these up in a bit.

Reply:
The natural alignment of a 64-bit pointer is 8 bytes. The `alignas` specifier cannot be used to weaken the natural alignment of the declaration to which it applies. If it was appearing unaligned for some reason, then it's because the structure as a whole was allocated unaligned, but this specifier can do nothing to correct that. Unless you've changed something, the last I knew con4m's gc allocation was always handing out 16-byte aligned pointers, so if you've got a struct that's not so aligned, I'd be looking for corruption somewhere

https://en.cppreference.com/w/c/language/_Alignas